src/physics/matter-js/components/Gravity.js

```diff
@@ -5,20 +5,20 @@
  */
 
 /**
- * [description]
+ * A component to manipulate gravity for Matter.js bodies.
  *
  * @name Phaser.Physics.Matter.Components.Gravity
  * @since 3.0.0
  */
 var Gravity = {
 
     /**
-     * [description]
+     * Ignore gravity.
```

Comment:
This is a bit too short for such an important method really. It should at least explain where the gravity comes from (the world simulation) and that it causes this body to ignore that completely. It can also be toggled in real-time.